<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>783</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>181</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Image</string>
    </property>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Load folder</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_R</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Calibration</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Find corner</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 Find intrinsic</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>1.3 Find extrinsic</string>
     </property>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>1.3 Find extrinsic</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.4 Find distortion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1.5 Show result</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Augmented Reality</string>
    </property>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Show words on board</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Show words vertical</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>171</width>
       <height>87</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Stereo disparity map</string>
    </property>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Stereo disparity map</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>4. SIFT</string>
    </property>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Keypoints</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Matched keypoints</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>5. VGG19</string>
    </property>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5.1 Show Augmented
Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5.2 Show Model
Structure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5.3 Show Acc
and Loss</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5.4 Inference</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>256</width>
       <height>201</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predict =</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
